ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-multi.yaml

```diff
@@ -0,0 +1,50 @@
+base_images:
+  upi-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.17"
+  multi-latest:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.17"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: node-density-heavy-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      COMPUTE_NODE_TYPE: Standard_D4s_v5
```

Comment:
@mehabhalodiya This config is pure amd64, no arm instances are created. Please refer to Sahail's PR https://github.com/openshift/release/pull/59678  for the 24 nodes. You need to add the following env and test. If you have questions about why, you can ping me or Sahil.
```
    env:
      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
      ADDITIONAL_WORKER_VM_TYPE: Standard_D4ps_v5
      ADDITIONAL_WORKERS: "3"
      COMPUTE_NODE_REPLICAS: "3"
      COMPUTE_NODE_TYPE: Standard_D4s_v5
      CONTROL_PLANE_INSTANCE_TYPE: Standard_D4s_v5
      CUSTOM_AZURE_REGION: eastus
      OCP_ARCH: amd64
      PROFILE_TYPE: reporting
      WORKER_REPLICA_COUNT: "24"
    test:
    - ref: ipi-install-heterogeneous
    - chain: workers-scale
  ```